Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_metamorphic.yml

```diff
@@ -2140,6 +2140,22 @@
     sprite: Objects/Consumable/Drinks/toxinsspecialglass.rsi
     state: icon
 
+- type: entity
+  parent: DrinkGlass
+  id: DrinkScrantonSpecialGlass
+  suffix: scranton special
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        maxVol: 30
+        reagents:
+        - ReagentId: ScrantonSpecial
+          Quantity: 30
+  - type: Icon
+    sprite: Objects/Consumable/Drinks/scrantonspecialglass.rsi
+    state: icon
```

Comment:
Entirely new prototypes should be put into a parallel file under a `_moffstation` directory, ie. `.../Resources/Prototypes/_Moffstation/Entities/Objects/Consumable/Drinks/drinks_metamorphic.yml`.